lib/gui/os/dialog/services/dialog.js

```diff
@@ -39,8 +39,9 @@ module.exports = function($q, SupportedFormatsModel) {
    * });
    */
   this.selectImage = () => {
+    const parent = electron.remote.getCurrentWindow();
```

Comment:
Can we move this line inside the promise, and rename `parent` to `currentWindow` (since otherwise reading `parent` at this point makes it look that the browser window returned by `getCurrentWindow()` is the parent of the current app)? For example:

```js
return $q((resolve, reject) => {
    const currentWindow = electron.remote.getCurrentWindow();
    electron.remote.dialog.showOpenDialog(currentWindow, {
    ...
```